core/include/drivers/stm32_rifsc.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause) */
+/*
+ * Copyright (c) 2020, STMicroelectronics
+ */
+#ifndef __STM32_RIFSC_H
+#define __STM32_RIFSC_H
+
+#include <types_ext.h>
+#include <util.h>
+
+struct risup_cfg {
```

Comment:
Are these 2 structures to be used outside _stm32_rifsc.c_?
If not, I would suggest to defined fine only in the scope of this source file.

Reply:
Right, I'll remove the header file